app/views/govuk_publishing_components/components/docs/step_by_step_nav_header.yml

```diff
@@ -26,3 +26,22 @@ examples:
       title: 'With a tracking id'
       path: '#'
       tracking_id: 'this-is-the-tracking-id'
+  with_custom_tracking:
+    data:
+      title: 'With a tracking id'
+      path: '#'
+      tracking_id: 'this-is-the-tracking-id'
+      tracking_category: "customTrackingCategoryClicked"
+      tracking_action: "customTrackingAction"
+      tracking_label: "customTrackingLabel"
+      tracking_dimension: "customTrackingDimension"
+      tracking_dimension_index: "23"
+  without_custom_dimension:
+    data:
+      title: 'With a tracking id'
+      path: '#'
+      tracking_id: 'this-is-the-tracking-id'
+      tracking_category: "customTrackingCategoryClicked"
+      tracking_action: "customTrackingAction"
+      tracking_label: "customTrackingLabel"
+      tracking_dimension: ""
```

Comment:
@andysellick @vanitabarrett - not really happy that I have to send an empty string. Is there a way around this?

Had to do this because of this https://github.com/alphagov/govuk_publishing_components/pull/1533/files#diff-b75e540e4545c561ea33a018910a7bf1R14

Reply:
That code looks like it should use the `title` by default if a `tracking_dimension` isn't passed?